akka-docs/src/main/paradox/includes/cluster.md

```diff
@@ -0,0 +1,65 @@
+<!--- #cluster-singleton --->
+### Cluster Singleton
+
+For some use cases it is convenient or necessary to ensure only one 
+actor of a certain type is running somewhere in the cluster.
+This can be implemented by subscribing to member events, but there are several corner
+cases to consider. Therefore, this specific use case is covered by the Cluster Singleton.
+
+<!--- #cluster-singleton --->
+
+<!--- #cluster-sharding --->
+### Cluster Sharding
+
+Distributes actors across several nodes in the cluster and supports interaction
+with the actors using their logical identifier, but without having to care about
+their physical location in the cluster.
+
+<!--- #cluster-sharding --->
+
+<!--- #cluster-ddata --->
+### Distributed Data
+
+*Akka Distributed Data* is useful when you need to share data between nodes in an
+Akka Cluster. The data is accessed with an actor providing a key-value store like API.
+
+<!--- #cluster-ddata --->
+
+<!--- #cluster-routers --->
+### Cluster Aware Routers
+
+All routers can be made aware of member nodes in the cluster, i.e.
+deploying new routees or looking up routees on nodes in the cluster.
+When a node becomes unreachable or leaves the cluster the routees of that node are
+automatically unregistered from the router. When new nodes join the cluster, additional
+routees are added to the router, according to the configuration.
```

Comment:
this text is not common for Classic and Typed. This only describes how Classic routers work.